### PR TITLE
Frontend integration

### PR DESCRIPTION
**Pull request by IhateTrains**
Integrated Fronter. It reports it can't find the converter's output folder but that's OK, the converter doesn't output anything yet.

### CODE REVIEW

File: ImperatorToCk3/Source/Configuration/Configuration.cpp [hunk 1]
```diff
@@ -25,32 +25,58 @@ Configuration::Configuration(std::istream& theStream)
...
+	registerKeyword("CK3directory", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString path(theStream);
+		Ck3Path = path.getString();
+		});
+	registerKeyword("output_name", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString nameStr(theStream);
+		outputName = nameStr.getString();
+		Log(LogLevel::Info) << "Output name set to: " << outputName;
+		});
+	registerKeyword("ImperatorDeJure", [this](const std::string& unused, std::istream& theStream) {
```
Comment (Zemurin) [inline ImperatorToCk3/Source/Configuration/Configuration.cpp:50]: Fronter will serve strings, not ints, for all configuration options. You need singleString and stoi after.

File: ImperatorToCk3Tests/CommonTests/VersionTests.cpp [hunk 2]
```diff
@@ -4,44 +4,44 @@
 
 TEST(ImperatorWorld_VersionTests, VersionHigherOrEqual)
 {
-	const Version version1 = Version("1.4.1");
-	const Version version2 = Version("1.3.2");
-	const Version requiredVersion = Version("1.3.2.0");
+	const Version version1("1.4.1");
```
Comment (Zemurin) [inline ImperatorToCk3Tests/CommonTests/VersionTests.cpp:7]: You can drop version as it was moved into commonItems/GameVersion

Comment (IhateTrains) [reply ImperatorToCk3Tests/CommonTests/VersionTests.cpp:7]: I wanted to do this in the next PR but can do now.

**Comment by IhateTrains** [comment]
I don't know what's broken in appveyor now 🤔

**Comment by IhateTrains** [comment]
Done ✔️

**Pull request merged**